src/supplemental/quic/msquic_dial.c

```diff
@@ -134,6 +134,16 @@ nni_quic_dialer_init(void **argp)
 	// multi_stream is disabled by default
 	d->enable_mltstrm = false;
 
+	memset(&d->settings, 0, sizeof(QUIC_SETTINGS));
+
+	d->qidle_timeout = 90;
```

Comment:
Use Marco instead of constant value is better.

Reply:
Okey fine...